@uportal/notification-modal/babel.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
     presets: ['@babel/preset-env'],
-    plugins: [['@babel/plugin-transform-runtime', { useESModules: true }]]
+    plugins: [
+        '@babel/plugin-transform-modules-commonjs',
```

Comment:
:thinking: Looking in `@babel/preset-env` this plugin appears to be included
https://github.com/babel/babel/blob/8ca99b9f0938daa6a7d91df81e612a1a24b09d98/packages/babel-preset-env/package.json#L43
Maybe the `modules` option needs to be set? https://babeljs.io/docs/en/next/babel-preset-env.html#modules